docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[data-streams]]
+= Data streams
+
+{agent} uses data streams to easily store append-only time series data across multiple indices
+while giving you a single named resource for requests.
+Data streams are well-suited for logs, metrics, traces, and other continuously generated data.
+They offer a host of benefits over other indexing strategies:
+
+* *Reduced number of fields per index*: Indices only need to store a specific subset of your
+data–meaning no more indices with hundreds of thousands of fields.
+This leads to better space efficiency and faster queries.
+As an added bonus, only relevant fields are shown in Discover.
+
+* *More granular data control*: For example, filesystem, load, cpu, network, and process metrics are sent
+to different indices–each potentially with its own rollover, retention, and security permissions.
+
+* *Flexible*: Use the custom namespace component to divide and organize data in a way that
+makes sense to your use-case or company.
+
+* *Fewer ingest permissions required*: Data ingestion only requires permissions to append data.
+
+[discrete]
+[[data-streams-naming-scheme]]
+== Data stream naming scheme
+
+{agent} uses the Elastic data stream naming scheme to name data streams.
+The naming scheme splits data into different streams based on the following components:
+
+`type`::
+A generic `type` describing the data, such as `logs`, `metrics`, `traces`, or `synthetics`.
+// Corresponds to the `data_stream.type` field.
+
+`dataset`::
+The `dataset` is defined by the integration and describes the ingested data and its structure for each index.
+For example, you might have a dataset for process metrics with a field describing whether the process is running or not,
+and another dataset for disk I/O metrics with a field describing the number of bytes read.
++
+For APM data, the `dataset` is the instrumented service's `service.name`.
+
+`namespace`::
+A user-configurable arbitrary grouping, such as an environment (`dev`, `prod`, or `qa`),
+a team, or a strategic business unit.
+A `namespace` can be up to 100 bytes in length (multi-byte characters will count toward this limit faster).
```

Comment:
```suggestion
A `namespace` can be up to 100 bytes in length (multibyte characters will count toward this limit faster).
```

Reply:
Both forms are correct, but I think all one word is in the spirit of Elastic style.